src/components/PageHeader/index.js

```diff
@@ -2,17 +2,17 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-const PageHeader = ({ title, description }) => (
+const PageHeader = ({ description, children }) => (
   <div className="coa-PageHeader">
-    <h1>{title}</h1>
+    <h1>{children}</h1>
     {description && (
       <p className="coa-PageHeader__description">{description}</p>
     )}
   </div>
 );
 
 PageHeader.propTypes = {
-  title: PropTypes.string.isRequired,
+  children: PropTypes.node.isRequired,
```

Comment:
is there any difference between `node` and `element`?

Reply:
element I think expects one react element, node can be anything (string, number, arrays, fragments, an element, etc)